RoktDemo/Views/SubViews/RoktEmbeddedSwiftUIView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  RoktEmbeddedSwiftUIView.swift
+//  RoktDemo
+//
+//  Copyright 2020 Rokt Pte Ltd
+//
+//  Licensed under the Rokt Software Development Kit (SDK) Terms of Use
+//  Version 2.0 (the "License");
+//
+//  You may not use this file except in compliance with the License.
+//
+//  You may obtain a copy of the License at https://rokt.com/sdk-license-2-0/
+
+import Rokt_Widget
+import SwiftUI
+
+struct RoktEmbeddedSwiftUIView: UIViewRepresentable {
```

Comment:
@nedarokt this is how we use UIKit inside of SwiftUI

Reply:
Cool, it's similar to Jetpack compose